### ISSUE TITLE
File preview WebDAV API

### ISSUE THREAD

**Issue by MarPiRK** [nextcloud/server#6176]
I was thinking about integration of NextCloud with Rocket.Chat webchat. From what I can see there is no standard API for thumbnails besides _nc:has-preview_ PROPFIND parameter. Having thumbnails available to GET from server would greatly reduce effort to integrate NextCloud to webchat, as webchat doesn't need to download entire file, try to generate thumbnail and it doesn't need to have thumbnail generators for different file types, which NextCloud has by itself.

So to sum up it would be great to have option to download pre-generated thumbnail via WebDAV API. I believe that NextCloud has some sort of thumbnail cache, so it would be just matter of exposing it in some way.

**Comment by MorrisJobke** [comment]
Duplicate of #3923
